### DISCUSSION TITLE
這項目還在發展嗎??

### DISCUSSION THREAD

**Discussion by lokpui** [jqhph/dcat-admin#1974] [General]
這項目真的蠻好用的, 也暫時能滿足要求, 但看到好像很久沒更新, 想知道下狀況 (期待著應用市場)

**Comment by ShrRung** [comment]
貌似要不更新了

**Reply by lanceye158** [reply]
我想要这里前端与后端谁能帮我做一个完整的

**Comment by p2p-sys** [comment]
項目替代方案是什麼？

**Reply by lanceye158** [reply]
我还有一套其他类型的源码谁需要告诉我

**Reply by zhanghongwuxiao** [reply]
> 我还有一套其他类型的源码谁需要告诉我

大哥，我要

**Comment by jinhai6668** [comment]
> 這項目真的蠻好用的, 也暫時能滿足要求, 但看到好像很久沒更新, 想知道下狀況 (期待著應用市場)

作者有自己的主页工作，你有什么需要扩展的，可能可以付费让他做
